tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py

```diff
@@ -93,3 +93,99 @@ def test_dot_invalid(is_module, xp, device, a_shape, b_shape, dtype):
         return xp.dot(a, b)
     else:
         return a.dot(b)
+
+
+class NumpyLinalgOpTest(op_utils.NumpyOpTest):
+
+    dodge_nondifferentiable = True
+    forward_accept_errors = (TypeError,
+                             chainerx.DtypeError)
```

Comment:
Let's not use accept errors just to work around fp16 parameterizations. How about being more explicit with fp32, fp64 (as you previously wrote). You could of course check the dtype and add logic to `setup` to skip in this base class but that'd be a bit cumbersome to maintain.

Reply:
Alright, I'll change that back.